noir-projects/aztec-nr/aztec/src/macros/mod.nr

```diff
@@ -141,8 +138,8 @@ comptime fn generate_contract_library_method_compute_note_hash_and_optionally_a_
 
         // Contracts that do define notes produce an if-else chain where `note_type_id` is matched against the
         // `get_note_type_id()` function of each note type that we know of, in order to identify the note type. Once we
-        // know it we call `aztec::note::utils::compute_note_hash_and_optionally_a_nullifier` (which is the one that
-        // actually does the work) with the correct `unpack()` function.
+        // know it we call we correct `unpack` method from the `Packable` trait to obtain the underlying note type, and
```

Comment:
typo?

Reply:
"we call we correct `unpack` method from the `Packable` trait to obtain the underlying note type"

Reply:
That seems correct?

Reply:
"we call we correct" confused me to the point where I blacked out.

Reply:
Also got short-circuited, lol